src/resources/postcss/views/skeleton/_single-event.pcss

```diff
@@ -382,8 +419,30 @@
 			margin-bottom: var(--spacer-3);
 			width: 100%;
 
+			/**
+			 * TODO: This is done specifically for IE11 and because we could not
+			 * touch templates. Use object-fit: cover; in the future if IE11
+			 * support gets dropped or update the HTML template if possible.
+			**/
```

Comment:
Soon hopefully!
https://make.wordpress.org/core/2021/03/04/discussion-dropping-support-for-ie11/

Reply:
oh yeah 🙌  `object-fit: cover` ftw